mlflow/sentence_transformers/__init__.py

```diff
@@ -397,6 +397,20 @@ class that describes the model's inputs and outputs. If not specified but an
     )
 
 
+def _update_load_kwargs():
```

Comment:
```suggestion
def _get_load_kwargs():
```

because this function updates nothing.